src/scripts/h5p-multi-media-choice-option.js

```diff
@@ -302,4 +485,12 @@ export class MultiMediaChoiceOption {
       }
     });
   }
+  /**
+   * Pauses the audio/video
+   */
+  pauseMedia()  {
+    if  (this.instance) {
+      this.instance.pause();
+    }
+  }
```

Comment:
What about 'k' key stroke ? ( I didn't test yet, but it has to work to play / pause)